crates/ide_completion/src/render.rs

```diff
@@ -531,6 +542,10 @@ fn main() { som$0 }
                         ),
                         lookup: "main",
                         detail: "-> ()",
+                        relevance: CompletionRelevance {
+                            exact_name_match: false,
+                            exact_type_match: true,
```

Comment:
I'm thinking unit type should never get the `exact_type_match` relevance bonus? It doesn't make sense to boost the relevance of a `fn() -> ()` inside of a `fn() -> ()`, because it is just as likely (maybe more likely) that the user wants a function with any other return type and just hasn't added the `;` yet.

Reply:
Seems reasonable!

Reply:
Done! I think this makes things a lot nicer.